docs/reference/search/search-your-data/learning-to-rank-model-training.asciidoc

```diff
@@ -0,0 +1,166 @@
+[[learning-to-rank-model-training]]
+=== Deploy and manage Learning To Rank models
+++++
+<titleabbrev>Deploy and manage LTR models</titleabbrev>
+++++
+
+preview::["The Learning To Rank feature is in technical preview and may be changed or removed in a future release. Elastic will work to fix any issues, but this feature is not subject to the support SLA of official GA features."]
```

Comment:
Any reason we don't want to just use the default copy here that comes with `preview::[]`?

Reply:
Just wanted to put the emphasis on the LTR feature.